hooks/post/30-upgrade.rb

```diff
@@ -1,6 +1,7 @@
 require 'fileutils'
 
 STEP_DIRECTORY = '/etc/foreman-installer/applied_hooks/post/'
+MONGO_REMOVAL_COMPLETE = '/etc/foreman-installer/.mongo_2_removed'.freeze
```

Comment:
We're starting to get more and more of these markers. I wonder if it's time for a `/var/lib/foreman-installer` for various state things, but let's leave that for a later stage.